ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/customercenter/navigation/CustomerCenterDestination.kt

```diff
@@ -0,0 +1,31 @@
+package com.revenuecat.purchases.ui.revenuecatui.customercenter.navigation
+
+import androidx.compose.runtime.Immutable
+import com.revenuecat.purchases.ui.revenuecatui.customercenter.data.FeedbackSurveyData
+import com.revenuecat.purchases.ui.revenuecatui.customercenter.data.PromotionalOfferData
+
+internal enum class CustomerCenterAnimationType {
+    SLIDE_HORIZONTAL,
+}
+
+@Immutable
+internal sealed class CustomerCenterDestination {
```

Comment:
Would it make sense to add the title property to the destination itself instead of the `CustomerCenterState`? Could make sense, so we remember to add it on the destination itself... However, it might be less flexible if we want to make it more dynamic, so not a strong opinion here.

Reply:
Yes! they come localized from the backend